### ISSUE TITLE
cya directories show excluded dirs and files

### ISSUE THREAD

**Issue by cspaniard** [cleverwise/cya#8]
The `directories` option is the closest we have to a "config-check" or a  "dry-run".

It would be nice to have `cya directories` show us the directories and files being excluded to be able to double check that we have the configuration file the way we need to, before we run our back up.

```
ACTION ⯮ Directories & Files Backing Up

Directories:
---------------------------------------------
/bin/ ... yes
/boot/ ... yes
[...]

Files:
---------------------------------------------
/etc/hosts ... yes
/etc/bash_completion.d/git-prompt ... yes
[...]

==============================================

Excluded Directories & Files

Directories:
---------------------------------------------
/usr/games/ ... excluded
[...]

Files:
---------------------------------------------
/usr/local/bin/search ... excluded
[...]

```

**Comment by cleverwise** [comment]
Good idea.

**Comment by cleverwise** [comment]
In 2.3.1+dev/2.4

**Comment by cspaniard** [comment]
There is a bug / inconsistency here...

The `configfile` command in the section

```
• Exclude subdirectories from backup:
You simply type EXCLUDE underscore then parent directory with slashes and between quotes add children directories separated by a space.  Now the subdirectores should NOT include the parent directory.  NO FORWARD SLASHES FOR CHILDREN DIRECTORIES!
```

According to this, the configuration line should start with the word `EXCLUDE_`, however it does not work mainly because `cya` is looking for `EXCLUDED_` (ended with a `D`)

```
   1273         # Get excluded data then display
   1274         EXCLUDED_SOURCE_FILE=`cat $CONFIG_FILE | grep EXCLUDED_`
```

Therefore nothing is shown...

**Comment by cleverwise** [comment]
You are correct; copy and paste can get one in trouble.  That isn't right.  The D should not be there and has been fixed in 2.3.2.

**Comment by cspaniard** [comment]
Works as expected
